app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -3,42 +3,16 @@ import { Button, PageLoader } from "neetoui";
 import EmptyState from "components/Common/EmptyState";
 import EmptyNotesListImage from "images/EmptyNotesList";
 import { Header, SubHeader } from "neetoui/layouts";
+import {
+  InitialNotesData,
+  TagsValuesArray,
+  DashboardPaginationProps,
+} from "./constants";
 
```

Comment:
Please move this along with the other relative imports. 
Please refer to [Sorting import statements](https://www.bigbinary.com/react-best-practices/sort-import-statements#sorting-import-statements) for more details.